src/block/Hopper.php

```diff
@@ -57,6 +57,17 @@ public function getStateBitmask() : int{
 		return 0b1111;
 	}
 
+	public function getFacing() : int{ return $this->facing; }
```

Comment:
PocketMine code style. One line per statement.

Reply:
Something is wrong? : /